core/server/master/src/main/java/alluxio/master/FaultTolerantAlluxioMasterProcess.java

```diff
@@ -135,14 +135,14 @@ private boolean gainPrimacy() throws Exception {
     LOG.info("Becoming a leader.");
     // Don't upgrade if this master's primacy is unstable.
     AtomicBoolean unstable = new AtomicBoolean(false);
-    try (Scoped scoped = mLeaderSelector.onStateChange(state -> unstable.set(true))) {
+    try (Scoped ignored = mLeaderSelector.onStateChange(state -> unstable.set(true))) {
       if (mLeaderSelector.getState() != State.PRIMARY) {
         LOG.info("Lost leadership while becoming a leader.");
         unstable.set(true);
       }
       stopMasters();
       LOG.info("Standby stopped");
-      try (Timer.Context ctx = MetricsSystem
+      try (Timer.Context ignored1 = MetricsSystem
```

Comment:
Ah yeah, case-in-point about needing `ignored1`, `ignored2`, etc. See my prior comment about this.

Reply:
Ditto.